src/index.html

```diff
@@ -11,7 +11,164 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>HTML Form</h1>
     <script type="text/javascript" src="./main.js"></script>
+    <form
+      action="https://mate-academy-form-lesson.herokuapp.com/create-application"
+      method="POST"
+      autocomplete="off"
+    >
+      <fieldset>
+        <legend>Personal information</legend>
+        <label for="surname" class="form-field-top">
+          Surname:
+          <input
+            type="text"
+            name="surname"
+            id="surname"
+          >
+        </label>
+        <label for="name" class="form-field">
+          Name:
+          <input
+            type="text"
+            name="name"
+            id="name"
+          >
+        </label>
+        <label for="age" class="form-field">
+          How old are You?
+          <input
+            type="number"
+            name="age"
+            id="age"
+            max="100"
+            min="1"
+            value="12"
+          >
+        </label>
+        <label for="dateOfBirth" class="form-field">
+          Full date of birth:
+          <input
+            type="date"
+            name="dateOfBirth"
+            id="dateOfBirth"
+          >
+        </label>
+        <label for="terms" class="form-field-bottom">
+          I accept the term of the agreement
+          <input
+            type="checkbox"
+            name="terms"
+            id="terms"
+          >
+        </label>
+      </fieldset>
+      <fieldset class="fieldset">
```

Comment:
```suggestion
      </fieldset>
      
      <fieldset class="fieldset">
```
make indents between blocks of code for better readability